client/me/get-apps/desktop-download-card.jsx

```diff
@@ -0,0 +1,215 @@
+					'{{thirdAvailableLink}}Mac{{/thirdAvailableLink}}.',
+					{ components: this.getTranslateComponents( platform ) }
+				);
+			default:
+				return translate( 'Also available for: ' +
+					'{{firstAvailableLink}}MacOS{{/firstAvailableLink}}, ' +
+					'{{secondAvailableLink}}Linux (.tar.gz){{/secondAvailableLink}}, ' +
+					'{{thirdAvailableLink}}Linux (.deb){{/thirdAvailableLink}}.',
+					{ components: this.getTranslateComponents( platform ) }
+				);
+		}
+	}
+
+	getButtonClickHandler( platform ) {
+		const {
+			trackWindowsClick,
+			trackMacClick,
+			trackLinuxTarClick,
+			trackLinuxDebClick,
+		} = this.props;
+
+		switch ( platform ) {
+			case 'MacIntel':
+				return trackMacClick;
+			case 'Linux i686':
+				return trackLinuxTarClick;
+			case 'Linux i686 on x86_64':
+				return trackLinuxDebClick;
+			default:
+				return trackWindowsClick;
+		}
+	}
+
+	getButtonLink( platform ) {
+		switch ( platform ) {
+			case 'MacIntel':
+				return MAC_LINK;
+			case 'Linux i686':
+			case 'Linux i686 on x86_64':
+				return LINUX_TAR_LINK;
+			default:
+				return WINDOWS_LINK;
+		}
+	}
+
+	getCardTitle( platform ) {
+		const { translate } = this.props;
+		switch ( platform ) {
+			case 'MacIntel':
+				return translate( 'Desktop App for Mac' );
+			case 'Linux i686':
+			case 'Linux i686 on x86_64':
+				return translate( 'Desktop App for Linux' );
+			default:
+				return translate( 'Desktop App for Windows' );
+
+		}
+	}
+
+	getLinkAnchorTag( platformLink ) {
+		const {
+			trackWindowsClick,
+			trackMacClick,
+			trackLinuxTarClick,
+			trackLinuxDebClick,
+		} = this.props;
+
+		switch ( platformLink ) {
+			case MAC_LINK:
+				return <a href={ platformLink } onClick={ trackMacClick } />;
+			case LINUX_TAR_LINK:
+				return <a href={ platformLink } onClick={ trackLinuxTarClick } />;
+			case LINUX_DEB_LINK:
+				return <a href={ platformLink } onClick={ trackLinuxDebClick } />;
+			default:
+				return <a href={ platformLink } onClick={ trackWindowsClick } />;
+		}
+	}
+
+	render() {
+		const { translate } = this.props;
+		const platform = ( navigator.platform && navigator.platform.length > 0 ) ? navigator.platform : false;
+		return (
+			<Card className="get-apps__desktop">
+				<div className="get-apps__card-text">
+					<h3 className="get-apps__card-title">{ this.getCardTitle( platform ) }</h3>
+					<p className="get-apps__description">{ this.getDescription( platform ) }</p>
+					<p className="get-apps__also-available">
+						{ this.getRequirementsText( platform ) }
+						{ this.getAlsoAvailableText( platform ) }
+					</p>
+				</div>
+				<Button
+					className="get-apps__desktop-button"
+					href={ this.getButtonLink( platform ) }
+					onClick={ this.getButtonClickHandler( platform ) }>
+					{ translate( 'Download' ) }
+				</Button>
+			</Card>
+		);
+	}
+}
+
+const mapDispatchToProps = {
+	trackWindowsClick: () => recordTracksEvent( 'calypso_app_download_windows_click' ),
```

Comment:
I guess I must have misread earlier. I'll help test again next week, choose a style that you like better. Maybe fat arrow since we explicitly ignore the onClick handler args.